src/github.com/matrix-org/dendrite/typingserver/api/output.go

```diff
@@ -16,16 +16,14 @@ package api
 type OutputTypingEvent struct {
 	// The Event for the typing edu event.
 	Event TypingEvent `json:"event"`
+	// Users typing in the room at the event.
+	TypingUsers []string `json:"typing_users"`
```

Comment:
Can you expand the `OutputTypingEvent` doc comment to explain why we're sending both a TypingEvent and list of currently typing users?